app/assets/locales/en.json

```diff
@@ -460,6 +465,12 @@
     "resend_activation_link": "If you haven't received an activation email or if you are having a problem using it, click on the button below to request a new activation email.",
     "resend_btn_lbl": "Resend Verification"
   },
+  "confirm_terms_page": {
+    "title": "Confirm terms and conditions",
+    "account_unconfirmed": "Your account has not accepted the terms and conditions.",
+    "message": "Please accept the terms and conditions for Greenlight.",
+    "confirm_btn_lbl": "Confirm"
```

Comment:
Is there no plain confirm button that we can reuse from somewhere else?

Reply:
Couldn't find one so I reused the resend button from the VerifyAccount page for a plain confirm.